test/types/community/cursor.test-d.ts

```diff
@@ -74,13 +74,13 @@ expectNotType<{ age: number }[]>(await typedCollection.find().project({ name: 1
 expectType<{ notExistingField: unknown }[]>(
   await typedCollection.find().project({ notExistingField: 1 }).toArray()
 );
-expectNotType<TypedDoc[]>(await typedCollection.find().project({ notExistingField: 1 }).toArray());
+expectType<TypedDoc[]>(await typedCollection.find().project({ notExistingField: 1 }).toArray());
```

Comment:
I thought this was supposed to yield Document[], which shouldn't match TypedDoc[]?

Reply:
Corrected the test, I think this is correct, unless we want to change the return to always be generic?